cmd/cluster/core/create.go

```diff
@@ -256,6 +257,17 @@ func createCommonFixture(ctx context.Context, opts *CreateOptions) (*apifixtures
 
 	}
 
+	// validate pausedUntil value
+	// valid values are either "true" or RFC3339 format date
+	if len(opts.PausedUntil) > 0 {
```

Comment:
Could we combine these two if statements into one?

Reply:
Yes